chart/templates/pgbouncer/pgbouncer-deployment.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.pgbouncer.replicas }}
```

Comment:
I understand that you have added the default value in the `values.yaml` file, but it may be good to have `{ ... | default 1}` here too, so that it will still work in the expected way even if the values file is messed up.

Reply:
Okay let me add defaults in template

Reply:
@XD-DENG, can you expand on "even if the values file is messed up" more? If the values file shipped with the chart is messed up, we are so far from having anything working that duplicating defaults around seems like extra noise to me. Am I missing something?